packages/salesforcedx-vscode-apex/test/unit/telemetry/telemetry.test.ts

```diff
@@ -77,4 +78,32 @@ describe('Telemetry', () => {
     };
     assert.calledWith(sendEvent, 'deactivationEvent', expectedData);
   });
+
+  it('Should send correct data format on sendApexLSPActivationEvent', async () => {
+    const telemetryService = TelemetryService.getInstance();
+    telemetryService.initializeService(reporter, true);
+
+    telemetryService.sendApexLSPActivationEvent([0, 50]);
+    assert.calledOnce(sendEvent);
+
+    const expectedData = {
+      extensionName: 'salesforcedx-vscode-apex',
+      startupTime: match.string
```

Comment:
Had to use matchers (https://sinonjs.org/releases/v2.3.6/matchers/) or the test would never pass.